hal/src/sercom/spi.rs

```diff
@@ -1267,6 +1267,13 @@ where
         self.config.as_mut().regs.disable();
         self.config
     }
+
+    /// Disable and then re-enable the SPI peripheral
+    #[inline]
+    pub fn reset(&mut self) {
+        self.config.as_mut().regs.disable();
+        self.config.as_mut().regs.enable();
+    }
```

Comment:
what's this for? it doesn't reset (ala SWRST), and it's not clear from the manual what if anything it does reset

Reply:
My hope is that it will reset the DATA register, so that if we've gotten an overflow fault, there's nothing in the data register to be sent after clearing the fault